ignite/metrics/classification_report.py

```diff
@@ -0,0 +1,63 @@
+from typing import Callable, Dict, List, Optional, Sequence, Union
+
+import torch
+
+from ignite.metrics.metric import reinit__is_reduced
+from ignite.metrics.metrics_lambda import MetricsLambda
+from ignite.metrics.precision import Precision
+from ignite.metrics.recall import Recall
+
+__all__ = ["ClassificationReport"]
+
+
+class ClassificationReport(MetricsLambda):
+    def __init__(
+        self,
+        output_dict: bool = False,
+        output_transform: Optional[Callable] = None,
+        device: Union[str, torch.device] = torch.device("cpu"),
+        labels: Optional[List[str]] = None,
+    ):
+        self.precision = Precision(average=False, device=device,)
+        self.recall = Recall(average=False, device=device,)
+        self.averaged_precision = Precision(average=True, device=device,)
+        self.averaged_recall = Recall(average=True, device=device,)
```

Comment:
Can't reuse previous definitions with metrics arithmetics ?
```python
self.averaged_precision = self.precision.mean()
```

SImilarly, we can define F1 or reuse already present `metrics.fbeta` implementation.

Reply:
Yep, `F1` score would be great !

Reply:
Great, will add them with docs and test cases. Thanks for the review!